### PR TITLE
initial commit

### PR DESCRIPTION
**Pull request by bakageddy**

### CODE REVIEW

File: embark [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+set completion-bg                   "#1E1C31"
+set completion-fg                   "#CBE3E7"
+set completion-group-bg             "#87DFEB"
+set completion-group-fg             "#2D2B40"
+set completion-highlight-bg         "#FFE6B3"
+set completion-highlight-fg         "#2D2B40"
+
+set index-bg                        "#1E1C31"
+set index-fg                        "#CBE3E7"
+set index-active-bg                 "#8A889D"
```
Comment (skbolton) [inline embark]: Lets change this color to #3E3859

File: embark [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+set index-active-fg                 "#2D2B40"
```
Comment (skbolton) [inline embark]: This should be able to stay as the original foreground color but double check after changing the background color.

File: embark [hunk 3]
```diff
@@ -0,0 +1,37 @@
+set notification-error-bg           "#1E1C31"
+set notification-error-fg           "#F48FB1"
+set notification-warning-bg         "#1E1c31"
+set notification-warning-fg         "#FFE6B3"
+set notification-bg                 "#1E1C31"
+set notification-fg                 "#CBE3E7"
+
+set completion-bg                   "#1E1C31"
+set completion-fg                   "#CBE3E7"
+set completion-group-bg             "#87DFEB"
```
Comment (skbolton) [inline embark]: Can we change this to #A1EFD3

**Review by skbolton — CHANGES_REQUESTED** [review]
Just a couple of color tweaks

File: embark (hunk 3 above)
Comment (bakageddy) [reply embark]: Sure that would look close to the original

File: embark (hunk 2 above)
Comment (bakageddy) [reply embark]: Let me check this...

**Comment by bakageddy** [comment]
Does it look good?

**Comment by skbolton** [comment]
Looks good to me! Time to get this thing merged in. Thanks for the contribution.

**Pull request merged**